### PR TITLE
To do - Astrid Persson

### PR DESCRIPTION
**Pull request by Aepera**
https://todo-aepera.netlify.app/

My reflections on how this week's project turned out:

Things I'd like to have clarified or explained in more detail:

### CODE REVIEW

File: code/src/components/AddTaskForm.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
+import React, { useState } from 'react'
+import uniqid from 'uniqid'
+import { useDispatch } from 'react-redux'
+
```
Comment (julialeonieros) [inline code/src/components/AddTaskForm.js:4]: Nice with blank line between outer and local imports, as well as spaces within curly brackets 👍

File: code/src/components/Header.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+const options = { weekday: 'long', month: 'long', day: 'numeric' };
+const date = new Date().toLocaleDateString("en-US", options)
+
+const Header = () => {
+
+  const items = useSelector((store) => store.tasks.items)
+  const total = items.length
+  const itemsDone = items.filter(task => task.isComplete === true)
+  // console.log(itemsDone.length)
```
Comment (julialeonieros) [inline code/src/components/Header.js]: Remember to remove console logs 😁

File: code/src/components/TaskList.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+            <div className='checkbox'>
+              <input
+                type='checkbox'
+                checked={task.isComplete}
+                onChange={() => dispatch(tasks.actions.toggleComplete(task.id))}      
+              />            
+            </div>
+
+          {/* <label for='text'><p>{task.text}</p></label> */}
```
Comment (julialeonieros) [inline code/src/components/TaskList.js]: ... and code that's been commented out

File: code/src/reducers/tasks.js [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+          return { ...task, isComplete: !task.isComplete}
+        } else {
+          return task
+        }
+      })
+      store.items = taskDoneList
+    },
+    deleteTask: (store, action) => {
+      const taskRemovedList = store.items.filter(task => task.id !== action.payload) 
+          store.items = taskRemovedList
```
Comment (julialeonieros) [inline code/src/reducers/tasks.js:27]: Nice that you use immutable approach to delete and complete tasks!

**Review by julialeonieros — COMMENTED** [review]
What a great app! I really like the design, very creative 👩‍🎨  The code is nice, clean and easy to read. Good naming of components and variables, it’s clear what they do and are used for 🙌

File: code/src/components/Header.js (hunk 2 above)
Comment (KarolCislowski) [reply code/src/components/Header.js]: busted 😸

File: code/src/components/TaskList.js (hunk 3 above)
Comment (KarolCislowski) [reply code/src/components/TaskList.js]: 👮🏻‍♂️

File: code/src/index.css [hunk 5]
```diff
@@ -1,13 +1,222 @@
...
 
 code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+h1 {
+  margin: 0;
+  color: #301a0c;
+  text-shadow: 1px 1px 1px #45526c5b;;
```
Comment (KarolCislowski) [inline code/src/index.css]: double semicolon here 😭

File: code/src/index.css [hunk 6]
```diff
@@ -1,13 +1,222 @@
...
+  background-color:#301a0c;
+  color: white;
+}
+
+.btn-all-delete {
+  background-color: #f8e1f5;
+  color: #301a0c;
+}
+
+/* CUSTOM CHECKBOX */
```
Comment (KarolCislowski) [inline code/src/index.css:170]: 😻

**Review by KarolCislowski — APPROVED** [review]
Great App 🌈 But please clean it from commented out parts of the code 🚔

**Review by KarolCislowski — APPROVED** [review]
Nice work!

**Pull request closed without merging**